app/services/report_importer.rb

```diff
@@ -59,9 +59,6 @@ def add_reporter_specific_data
           scanner.add_reporter_data(report, raw)
           break
         end
-      else
```

Comment:
You can also drop the `if scanner.respond_to?(:identify_origin)` condition since they should all support it already

Reply:
done